bin/.gitignore

```diff
@@ -0,0 +1,3 @@
+/main/
+/test/
+/default/
```

Comment:
This shouldn't be a thing. Please remove this file.

Reply:
Damn, how did that get in there. Sorry about that - should be gone now.